### PR TITLE
Reduce the number of required ENVs  in CI

### PR DESCRIPTION
**Pull request by michaljurecko**
Part of: https://keboola.atlassian.net/browse/COM-1058

Changes:
- Removed little used ENV variables.
- To easier setup of the Windows (parallel) testing.

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -1,6 +1,3 @@
 TEST_PROJECT_ID=
-TEST_PROJECT_NAME=
```
Comment (webrouse) [inline .env.dist:2]: Chcem spustat testy na Windows paralelne.
K tomu potrebujem oddeleny projekt.
Ako prvy krok -> zredukujem pocet potrebnych ENV premennych, aby som nemusel toho vela definovat duplicitne.

Preto som odstranil tieto 3 premenne.

File: internal/pkg/cli/create_test.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ func TestInteractiveCreateConfig(t *testing.T) {
 
 	// Init prompt and cmd
 	root := newTestRootCommandWithTty(c.Tty())
-	root.cmd.SetArgs([]string{"create", "--storage-api-token", testhelper.TestTokenMaster()})
+	root.cmd.SetArgs([]string{"create", "--storage-api-token", testhelper.TestToken()})
```
Comment (webrouse) [inline internal/pkg/cli/create_test.go:35]: Pred tym sme mali obycajny token (iba pre jeden test!) a vsade inde sa pouzival master token -> nechal som uz iba master token.

File: internal/pkg/remote/token_test.go [hunk 3]
```diff
@@ -31,7 +31,7 @@ func TestGetToken(t *testing.T) {
 	assert.Regexp(t, `DEBUG  HTTP      GET https://.*/v2/storage/tokens/verify | 200 | .*`, logs.String())
 	assert.Equal(t, tokenValue, token.Token)
 	assert.Equal(t, testhelper.TestProjectId(), token.ProjectId())
-	assert.Equal(t, testhelper.TestProjectName(), token.ProjectName())
+	assert.NotEmpty(t, token.ProjectName())
```
Comment (webrouse) [inline internal/pkg/remote/token_test.go:34]: Project name sa tiez pouzivalo iba v jednom test -> test som upravil.

File: internal/pkg/remote/token_test.go [hunk 4]
```diff
@@ -57,15 +57,3 @@ func TestGetTokenInvalid(t *testing.T) {
 	assert.Equal(t, 401, apiErr.HttpStatus())
 	assert.Nil(t, token)
 }
-
-func TestGetTokenExpired(t *testing.T) {
```
Comment (webrouse) [inline internal/pkg/remote/token_test.go:61]: Tento test pouzival expirovany token, ... test je vlastne duplicitny, pretoze APi nerozlisuje, ci je token expirovany, alebo neexistuje, vid `TestGetTokenInvalid`.

File: internal/pkg/cli/create_test.go (hunk 2 above)
Comment (webrouse) [reply internal/pkg/cli/create_test.go:35]: Tj. metoda `TestTokenMaster` -> premenovana na `TestToken`.

File: test/init-not-master-token/expected-stderr [hunk 5]
```diff
@@ -1 +0,0 @@
-Error: required master token, but the given token is not master
```
Comment (webrouse) [inline test/init-not-master-token/expected-stderr:1]: Zmazal som tento funkcionalny test, ak by user pouzil nie-master token.
Je vacsi prinos spustat testy paralelne, ako testovat tento edge-case.

**Review by kacurez — APPROVED** [review]
lgtm
